docs/source/api/tracker.md

```diff
@@ -81,45 +81,33 @@ is automatically stopped and won't be rerun.
 
 ### Tracker.autorun and async callbacks
 `Tracker.autorun` can accept an `async` callback function.  
-However, to make the async call reactive, you should wrap your async function in
-a `Tracker.withComputation` call.
+ To preserve reactivity for the reactive variables inside the async callback function, you must use a `Tracker.withComputation` call as described below:
+
+{% apibox "Tracker.withComputation" %}
 
 ```javascript
 Tracker.autorun(async function example1(computation) {
-  let asyncData = await asyncDataFunction();
-  let users =
-    await Tracker.withComputation(computation, () => Meteor.users.find({}).fetch());
-});
-```
-> If you want to get computation in other way you can use `Tracker.currentComputation`
-
-
-{% apibox "Tracker.withComputation" %}
+	// Code before the first await will stay reactive.
+	reactiveVar1.get() // will trigger rerun
```

Comment:
Please keep the comment style consistent - add semicolons, start with an upper letter, and end with a dot.